tools/fs/files.js

```diff
@@ -695,19 +696,154 @@ files.extractTarGz = function (buffer, destPath, options) {
   var tempDir = files.pathJoin(parentDir, '.tmp' + utils.randomToken());
   files.mkdir_p(tempDir);
 
+  if (! _.has(options, "verbose")) {
+    options.verbose = require("../console/console.js").Console.verbose;
```

Comment:
Note that `meteor run --verbose` will cause all the output from the `tar` and/or `7z.exe` commands to be displayed in the console, in case they don't seem to be working and we need to get some debugging clues.